style.css

```diff
@@ -340,14 +345,99 @@ ul {
   font-size: 1rem;
 }
 
+/* Pop-up for mobile */
+.modalInfo {
+  display: none;
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  margin: 0 auto;
+  z-index: 500;
+  overflow: auto;
+  background-color: rgb(25, 23, 101, 0.6);
+  backdrop-filter: blur(25px);
+  transition: all 1.5s ease-in;
+}
+
+.modal-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  height: 100vh;
+  margin: 45px auto;
+  background: rgba(255, 255, 255, 0.161);
+  padding: 1.5rem;
+  color: #fff;
+  text-align: center;
+  border: 1px solid #fff;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! There are some design errors in the `top` and `bottom` of the mobile version of the modal, please fix them. In the pictures below I have highlighted the differences between your design and [Figma's design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)!

![image](https://user-images.githubusercontent.com/63915024/189215445-1fc35d73-5d29-41d9-a45e-7a7cb33f66d8.png)